test/unit/web_page_media_card.spec.js

```diff
@@ -55,7 +55,7 @@ describe('<WebPageMediaCard />', () => {
       dbid: 26,
       quote: null,
       published: '1508470916',
-      archived: false,
+      archived: 0,
```

Comment:
Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

Reply:
Fixed by using CheckArchivedFlags constants